Playgrounds/1a_I_See_No_Nil.playground/contents.swift

```diff
@@ -146,7 +146,7 @@ For example the same does not apply to other ‘containers’, such as **Array**
 
 Whereas the following is acceptable:
 
-    let x:Int? = 8 // result is: Optional<Int>.Some(8)
+    let x:Int? = 8 // result is: Optional<Int>.some(8)
 
 * * *
 
```

Comment:
the whole "Is nil less than zero?" is no longer relevant due to [SE-0121 "Remove Optional Comparison Operators"](https://github.com/apple/swift-evolution/blob/master/proposals/0121-remove-optional-comparison-operators.md)